cpg_utils/constants.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+GCLOUD_ACTIVATE_AUTH_BASE = [
+    'gcloud',
+    '-q',
+    'auth',
+    'activate-service-account',
+    '--key-file=/gsa-key/key.json',
+]
+GCLOUD_ACTIVATE_AUTH = ' '.join(GCLOUD_ACTIVATE_AUTH_BASE)
+
+DEFAULT_GITHUB_ORGANISATION = 'populationgenomics'
+
+DEFAULT_CROMWELL_URL = 'https://cromwell.populationgenomics.org.au'
+DEFAULT_CROMWELL_AUDIENCE = (
+    '717631777761-ec4u8pffntsekut9kef58hts126v7usl.apps.googleusercontent.com'
+)
+
+
+CROMWELL_AUDIENCE = os.getenv('CROMWELL_AUDIENCE', DEFAULT_CROMWELL_AUDIENCE)
+CROMWELL_URL = os.getenv('CROMWELL_URL', DEFAULT_CROMWELL_URL)
+
+
+class AnsiiColors:
```

Comment:
Where is this used, and is this an opportunity to fix the spelling to `AnsiColors`?

(Or even `AnsiColours`! 😝 )